src/scales/scale.time.js

```diff
@@ -35,10 +39,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
+			source: 'auto'    // 'auto|labels'
 		}
 	};
 
+	function buildLookupTable(ticks, min, max, linear) {
```

Comment:
I think that the whole lookup (table) logic + `ticks.mode/source` options could be the base implementation for the `linear`, `radiallinear`, `log` and `time` scales (`numeric scale`?). This would refactor lot of code and make more consistent API between all that scales.